spring-cloud-starter-stream-sink-task-launcher-kubernetes/README.adoc

```diff
@@ -0,0 +1,68 @@
+//tag::ref-doc[]
+= TaskLauncher Kubernetes Sink
+
+The TaskLauncher Kubernetes sink consumes a `TaskLaunchRequest` message, then
+deploys and launches a task onto a Kubernetes instance using
+the Spring Cloud Deployer for Kubernetes.  With that being said a single
+TaskLauncher-CloudFoundry can launch different
+tasks based on the artifact that is specified in the `TaskLaunchRequest`.
+
+== Input
+
+=== Headers:
+
+* `Content-Type: application/octet-stream`
+
+=== Payload:
+
+A  byte array containing the `TaskLaunchRequest`
+
+== Output
+
+N/A (deploys and launches task to a Cloud Foundry instance).
+
+== Using the TaskLauncher
+
+The task launcher deploys and launches a
+link:https://cloud.spring.io/spring-cloud-task[Spring Cloud Task] Boot
+application as a task on the Kubernetes Platform based on contents from the
+`TaskLaunchRequest` message that it receives. The `TaskLaunchRequest` contains
+attributes that are used by the task launcher to execute the task:
+
+* uri - contains the uri to the Spring Cloud Task Boot application artifact
+that the task launcher will deploy to the Cloud Foundry Platform.
```

Comment:
May add a mention that it needs to be a docker resoure.  I can add this on merge).

Reply:
Yes, that would be good